examples/pytorch/IterativePruning/conda.yaml

```diff
@@ -0,0 +1,15 @@
+channels:
+- defaults
+- conda-forge
+- pytorch
+dependencies:
+- python=3.8.2
+- pytorch>=1.6.0
+- pip
+- pip:
```

Comment:
@ankan94 Please add the prettytable and ax-platform.

Reply:
Added